cfgov/jinja2/v1/_includes/ask/title-spanish.html

```diff
@@ -1,7 +1,5 @@
-{% if search_page %}
-    <title>Buscar por '{{ page_title }}' > Oficina para la Protección Financiera del Consumidor</title>
-    <meta name="title" content="Buscar por '{{ page_title }}' > Oficina para la Protección Financiera del Consumidor">
-{% else %}
-    <title>{{ page_title }} > Oficina para la Protección Financiera del Consumidor</title>
-    <meta name="title" content="{{ page_title }} > Oficina para la Protección Financiera del Consumidor">
-{% endif %}
\ No newline at end of file
+{%- if search_page -%}
```

Comment:
Eliminate the use of an include for this, just have the logic in `es-base` or `spanish-ask-base`.